gradle.properties

```diff
@@ -18,6 +18,7 @@
 org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.configureondemand=true
+org.gradle.java.installations.auto-download=false
```

Comment:
This change is not strictly required.

It disables Gradle from automatically downloading of JDK versions for use for building. When disabled all of the required Java toolchain must be present in the environment, which is what we expect.

Reply:
Should we set the same property for `docs` subproject?

Reply:
yes, I would argue for setting it in all of our gradle.properties files to prevent downloading and building with random JDK installations.